plugins/woocommerce-admin/client/marketing/components/PluginCardBody/PluginCardBody.tsx

```diff
@@ -0,0 +1,47 @@
+/**
+ * Internal dependencies
+ */
+import { CardBody } from '~/marketing/components';
+import './PluginCardBody.scss';
+
+type PluginCardBodyProps = {
+	icon: JSX.Element;
+	name: string;
+
+	/**
+	 * WooCommerce's Pill component to be rendered beside the name.
+	 */
+	pills?: Array< JSX.Element >;
+
+	description: string;
+	button: JSX.Element | null;
```

Comment:
I think it would be better to set button as optional and then set `button = null` in the props. That way we can omit button prop if it's not needed. Othwersie we are forcing the client to set button prop as null in case is not necessary.

Reply:
Good idea! I have made the change in [`88b0c9b` (#34415)](https://github.com/woocommerce/woocommerce/pull/34415/commits/88b0c9bf5c73206f85209a76f932497b32ed4ec6), but instead of `null`, I allow it to be `undefined` which IMO is more succinct.